coordinator-service/src/signed-data.ts

```diff
@@ -0,0 +1,30 @@
+import Joi from 'joi'
+
+import { logger } from './logger'
+
+export const signatureWith0xLength = 132
+export class SignedData {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    data: any
+    signature: string
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isSignedData(body: any): body is SignedData {
```

Comment:
I might use something like `body: {}` or maybe `body: Object` here so you don't have all the warnings

Reply:
I used `any` in a lot of places where `object` was a better fit. I changed it all over now.